src/Linters/BaseLinter.php

```diff
@@ -28,4 +28,26 @@ public function __construct(
   final public function getFile(): string {
     return $this->file;
   }
+
+  // A simple name for the linter, based on the class name
+  <<__Memoize>>
+  public function getLinterName(): string {
+    $r = new \ReflectionClass($this);
+    return $r->getShortName();
+  }
+
+  /**
+   * A user can choose to ignore all errors reported by this linter for a whole file using this string as a marker
+   */
+  protected function markerIgnoreAll(): string {
+    return 'HHAST_IGNORE_ALL['.$this->getLinterName().']';
+  }
+
+  /**
+   * A user can choose to ignore a specific error reported by this linter using this string as a marker
+   */
+  protected function markerFixMe(): string {
+    return 'HHAST_FIXME['.$this->getLinterName().']';
```

Comment:
it would be good to support both HHAST_FIXME and HHAST_IGNORE_ERROR - HH_FIXME should be used where it's expected that it will actually be fixed at some point, HH_IGNORE_ERROR when not.